### ISSUE TITLE
ModuleNotFoundError on Linux System

### ISSUE THREAD

**Issue by hybridpicker** [kbytesys/django-recaptcha3#12]
Getting this error while coverage run and deployment on my linux-server:

`App 3430 output: django.template.exceptions.TemplateSyntaxError: 'recaptcha3' is not a registered tag library. Must be one of:
App 14170 output: ModuleNotFoundError: No module named 'snowpenguin'`

Is there a other way to install that app in the settings?

I did like in the ReadMe in my settings.py:

`'snowpenguin.django.recaptcha3',`

**Comment by kbytesys** [comment]
How did you installed django-recaptcha3?  

Do you have this issue only when you run coverage or also when you start your project in development?

**Comment by hybridpicker** [comment]
I installed it with pip-command.
I have the issue also in development.
Have you tested it on CentOS?

**Comment by kbytesys** [comment]
Yes, I've tested the package with the latest centos using a venv as usual in many projects I done.

Which centos release do you use? Which Python version? Could you provide me a demo project?

Thanks!
